.circleci/config.yml

```diff
@@ -150,8 +150,9 @@ jobs:
       - run:
           name: Check TAG file
           command: |
-            c="$(./tag_exists ceph)"
-            if [ "$c" = ng ]; then
+            ceph_tag_exists="$(./tag_exists ceph)"
+            cephcsi_tag_exists="$(./tag_exists cephcsi)"
+            if [ "$ceph_tag_exists" = ng -o "$cephcsi_tag_exists" = ng ]; then
```

Comment:
I prefer using `||` operator instead of `-o`.
See manpage.
>NOTE: Binary -a and -o are inherently ambiguous.  Use 'test EXPR1 && test EXPR2' or  'test
>      EXPR1 || test EXPR2' instead.

https://manpages.ubuntu.com/manpages/hirsute/en/man1/test.1.html

Reply:
BTW, why ceph build is needed when cephcsi image is not found?

Reply:
>I prefer using || operator instead of -o.

I replaced `-o` with `||`.

Reply:
>BTW, why ceph build is needed when cephcsi image is not found?

That is the most complicated point.
Shortly speaking, Ceph build is required to resolve the dependencies between ceph-related packages.

First of all, the upstream cephcsi container is built from the ceph official image.
To completely stop depending on the container registries hosted by other organizations than Cybozu, our cephcsi container should be created from our self-built ceph image.
However, because our ceph image contains a relatively new `librados2` package and installation of the `librados-dev` package via `apt-get` requires an older version of `librados2`, the installation of `librados-dev` fails. To resolve this problem, I came up with installing librados-dev from deb package which is generated by ceph build.

Reply:
Understood.